packages/mocks/package.json

```diff
@@ -4,23 +4,17 @@
   "private": true,
   "repository": "git@github.com:kubev2v/forklift-console-plugin.git",
   "license": "Apache-2.0",
-  "main": "./src/cluster-proxy-server/server.ts",
+  "main": "./src/index.ts",
   "scripts": {
     "postinstall": "npx msw init ./generated --save",
     "clean": "rimraf ./dist ./coverage",
     "clean:all": "npm run clean -- ./node_modules",
     "build": "npm run clean && tsc",
-    "start": "ts-node .",
     "lint": "eslint . ",
     "lint:fix": "eslint . --fix"
   },
   "dependencies": {
-    "@mswjs/http-middleware": "^0.8.0",
-    "express": "^4.18.2",
-    "express-http-proxy": "^1.6.3",
-    "minimist": "^1.2.8",
-    "msw": "^1.2.1",
-    "process": "^0.11.10"
+    "msw": "^1.2.1"
```

Comment:
`msw` is used for ?

Reply:
creating service runner in postinstall (see above)